templates/docs/examples/patterns/tables/table-mobile-card-dark.html

```diff
@@ -0,0 +1,142 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Table / Mobile card Dark{% endblock %}
+
+{% block standalone_css %}patterns_table-mobile-card{% endblock %}
+
+{% set is_dark = true %}
+{% block content %}
+<table class="p-table--mobile-card" aria-label="Example of table transforming into mobile cards on small screens">
```

Comment:
Because these dark examples are practically all the same as light examples (the only difference being class on body) it would be good to extract the html of the table itself to some partial `_table-mobile-card.html` and then include it in both dark and light examples, to avoid dupliaction.

Like we have here in notifications: https://github.com/canonical/vanilla-framework/blob/main/templates/docs/examples/patterns/notifications/variants-dark.html

Reply:
Thanks for pointing it out